### PR TITLE
[luci/import] Add Unique operation

### PR DESCRIPTION
**Pull request by dr-venkman**
This commit will add unique operation to luci import

ONE-DCO-1.0-Signed-off-by: venkat.iyer <venkat.iyer@samsung.com>

### CODE REVIEW

File: compiler/luci/import/src/Nodes/CircleUnique.cpp [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+  // Create CircleUnique
+  auto node = graph->nodes()->create<CircleUnique>();
+  node->input(input_nodes[0]);
+
+  const auto *options = op.builtin_options.AsUniqueOptions();
+  node->output_type(luci_datatype(options->idx_out_type));
+
+  assert(outputs.size() > 0);
+  assert(int32_t(outputs.size()) == 2);
```
Comment (mhs4670go) [inline compiler/luci/import/src/Nodes/CircleUnique.cpp:62]: (trivial) `assert(outputs.size() > 0);` seems to be unnecesary.

File: compiler/luci/import/src/GraphBuilderRegistry.cpp [hunk 2]
```diff
@@ -129,6 +129,7 @@ GraphBuilderRegistry::GraphBuilderRegistry()
   CIRCLE_NODE(TOPK_V2, CircleTopKV2GraphBuilder);                                          // 48
   CIRCLE_NODE(TRANSPOSE, CircleTransposeGraphBuilder);                                     // 39
   CIRCLE_NODE(TRANSPOSE_CONV, CircleTransposeConvGraphBuilder);                            // 67
+  CIRCLE_NODE(UNIQUE, CircleUniqueGraphBuilder);                                           // 103
```
Comment (seanshpark) [inline compiler/luci/import/src/GraphBuilderRegistry.cpp:132]: Please remove below `// BuiltinOperator_UNIQUE = 103,` line

File: compiler/luci/import/src/Nodes/CircleUnique.cpp (hunk 1 above)
Comment (dr-venkman) [reply compiler/luci/import/src/Nodes/CircleUnique.cpp]: Removed the line, thanks.

File: compiler/luci/import/src/GraphBuilderRegistry.cpp (hunk 2 above)
Comment (dr-venkman) [reply compiler/luci/import/src/GraphBuilderRegistry.cpp:132]: Removed the line, thanks.

File: compiler/luci/import/src/Nodes/CircleUnique.cpp [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+#include <loco.h>
+
+namespace luci
+{
+
+bool CircleUniqueGraphBuilder::validate(const ValidateArgs &args) const
+{
+  if (args.op.inputs.size() != 1)
+    return false;
+
```
Comment (seanshpark) [inline compiler/luci/import/src/Nodes/CircleUnique.cpp:31]: please add `outputs.size() != 2` check too.

Comment (dr-venkman) [reply compiler/luci/import/src/Nodes/CircleUnique.cpp:31]: Alright, I have a question though, shouldn't the check be `outputs.size() != 1`  instead, considering that `Unique` generates two outputs by definition? Please let me know.

Comment (dr-venkman) [reply compiler/luci/import/src/Nodes/CircleUnique.cpp:31]: Added the check, thanks.

**Pull request merged**